Add draft for contributing guidelines (#3)
inbo/coding-club: .github/CONTRIBUTING.md

[repo]: https://github.com/inbo/coding-club
[issues]: https://github.com/inbo/coding-club/issues
[new_issue]: https://github.com/inbo/coding-club/issues/new
[website]: https://inbo.github.io/coding-club/
[email]: mailto:stijn.vanhoey@inbo.be

## Code of conduct

Please note that this project is released with a [Contributor Code of Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree to abide by its terms.

## How you can contribute

There are several ways you can contribute to this project.

### Share the love ❤️

Think coding club is a useful initiative? Let others discover it, by telling them in person, via Twitter or a blog post.

### Propose an idea 💡

Have an idea to improve the content or organization of the coding club? You have an interesting topic to tackle? Take a look at the [issue list][issues] to see if it isn't included or suggested yet. If not, suggest your idea as an [issue on GitHub][new_issue]. While we can't promise to include your idea, it helps to:

* Explain in detail how it would work.
* Keep the scope as narrow as possible.
* Provide some ideas for challenges to include.

### Improve the documentation 📖

> Noticed a typo on the website? You had problems to get started or setup? Good documentation makes all the difference, so your help to improve it is very welcome! [This website][website] is generated and hosted by GitHub by the files and folders inside the `docs` folder. Hence, By editing the markdown pages in the `docs` folder , you can directly propose improvements. If things did not work out, [report an issue][new_issue] and we can point you in the right direction.

Reviewer: ```suggestion
Noticed a typo on the website? Encountered problems getting started? Good documentation makes all the difference, so your help to improve it is very welcome! [This website][website] is generated and hosted by GitHub by the files and folders inside the `docs` folder. Hence, by editing the markdown pages in the `docs` folder , you can directly propose improvements. If things did not work out, [report an issue][new_issue] and we can point you in the right direction.
```

## Acknowledgements

These contributing guidelines are based on the `CONTRIBUTING.md` template for R packages by [Peter Desmet](https://gist.github.com/peterdesmet/e90a1b0dc17af6c12daf6e8b2f044e7c).

> 

Reviewer: ```suggestion
```
